pkg/sdk/testint/functions_integration_test.go

```diff
@@ -34,6 +34,7 @@ import (
 // TODO [SNOW-1348103]: test secure
 // TODO [SNOW-1348103]: python aggregate func (100357 (P0000): Could not find accumulate method in function CVVEMHIT_06547800_08D6_DBCA_1AC7_5E422AFF8B39 with handler dump)
 // TODO [SNOW-1348103]: add test with multiple imports
+// TODO [this PR]: test with multiple external access integrations and secrets
```

Comment:
Next PR or is it already done?

Reply:
it's tested on the resource level, so we are fine but I would also like to add it here (so yeah, one of the next PRs - will add ticket too)